components/licensor/typescript/ee/src/api.ts

```diff
@@ -31,6 +31,14 @@ export interface LicensePayload {
     seats: number
 }
 
+export enum LicenseSubscriptionLevel {
```

Comment:
Should there be a duplicate of `community` and `prod`?

Reply:
`CommunityLicense` and `ProfessionalLicense` were introduced[ in the licensor code](https://github.com/gitpod-io/gitpod/pull/9343) for the `Gitpod` license levels. This duplication exists now, since I was trying to standardize the license level names and didn't want to change the replicated code too much. We can clean it up in the future, right now this is a requirement of the existing code.

Reply:
Thanks for the quick fix, Nandaja. :partying_face: 

I know, it's less convenient, but I think having 2 separate pull requests (one PR per concern) would help us in general. In this specific case, the discussion about the LicenseSubscriptionLevel blocks merging the build fix. What do you think? :thinking:

Reply:
@corneliusludmann I agree with one issue per PR comment. Unfortunately here, if we force the build, it will update `api.ts`(this file is auto-generated by `genapi`). Meaning that people will have a change to be staged in their workspace which is not very nice. This is a mistake on my part, not including it in the license UI PR. But the fact that changes in this PR regenerates `api.ts` makes this change related to the main issue.